release_files/Dockerfile.ubuntu

```diff
@@ -153,6 +153,17 @@ RUN	apt-get update -y ; \
 
 RUN apt-get update && apt-get install -y --no-install-recommends libxml2 && rm -rf /var/lib/apt/lists/* && rm -rf /tmp/*
 
+# for HDDL
```

Comment:
No changes in Redhat dockerfile are needed?

Reply:
Correct me if I'm wrong but we never supported vpu for redhat